docs-beta/src/components/Table.js

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+import "../css/tailwind-styles.css"; 
+
+export default function Table() {
+    return(
+        <table class="table-fixed">
+            <thead class="sticky top-0">
+                <tr>
+                    <th></th>
+                    <th>Lage</th>
```

Comment:
Why don't you create a little object that holds these in a JSON, and then feed it into the render, this way we can maintain this list better!

```json
{
  "Task Runners": [
    { "name": "lage", "link": "some url" }, 
    ...
  ],
  "Local computation caching": [

  ]
}
```